### ISSUE TITLE
version reported wrongly in webapp's title and menu

### ISSUE THREAD

**Issue by kayceegb** [Offerel/SyncMarks-Webapp#45]
Hi,

When you open the webapp, the version reported in page title reports as 2.17 instead of 1.2.17. The same happens when you open the webapp's menu. 

Is this desired ? 

K.

**Comment by Offerel** [comment]
Cant reproduce this...

**Comment by kayceegb** [comment]
It should be related to how open operating systems represent a new line in a file compared to windows. 
For windows it is a \r\n and in open systems it is just \n. 

If I reduce the offset in explode to 20 instead of 22, version have the good value.

**Comment by Offerel** [comment]
This has nothing to do with the OS. If you have not changed the file, or at least not the file endings, the version will be extracted the right way. The original file is UTF-8 with CRLF. It works correctly with my standard dev server running Debian 10 and also on a lamp system on a Windows 10 machine.

**Comment by kayceegb** [comment]
I have no reason to touch changelog.md. 
I'm in sync with what git have pulled and I don't see the same thing. 

$ git log -3
commit d7018a71ed9d2a3ca34b8a07724f3459d0db6933 (HEAD -> dev, origin/dev)
Author: Offerel <14041522+Offerel@users.noreply.github.com>
Date:   Sat Dec 12 20:22:01 2020 +0100

    Fix XHR response, close #43

commit d393d3e5df60ff34726937297c36713f33450c1f
Author: Offerel <14041522+Offerel@users.noreply.github.com>
Date:   Sat Dec 12 13:06:47 2020 +0100

    Fix for folder selection

commit ec317a02fedf87ab5e625919dfcfd648135f9001
Author: Offerel <14041522+Offerel@users.noreply.github.com>
Date:   Fri Dec 11 19:54:56 2020 +0100

    fixes #40
$ git diff
$ file changelog.md 
changelog.md: ASCII text
$ head -c 15 changelog.md | shed -
```
offset            asc hex dec oct bin
0000000000000000:  C  43  067 103 01000011
0000000000000001:  h  68  104 150 01101000
0000000000000002:  a  61  097 141 01100001
0000000000000003:  n  6E  110 156 01101110
0000000000000004:  g  67  103 147 01100111
0000000000000005:  e  65  101 145 01100101
0000000000000006:  L  4C  076 114 01001100
0000000000000007:  o  6F  111 157 01101111
0000000000000008:  g  67  103 147 01100111
0000000000000009:     0A  010 012 00001010
000000000000000A:  =  3D  061 075 00111101
000000000000000B:  =  3D  061 075 00111101
000000000000000C:  =  3D  061 075 00111101
000000000000000D:  =  3D  061 075 00111101
000000000000000E:  =  3D  061 075 00111101
```

So there is a gap with the way you get/set the version for this.
